tests/modules/layers/test_normalizations.py

```diff
@@ -20,3 +26,38 @@ def test_fp32groupnorm():
     norm = Fp32GroupNorm(2, 4)
     output = norm(x)
     assert output.dtype == torch.float16
+
+
+def test_rms_norm_core_algo():
```

Comment:
nit: rename to test_rms_norm

Reply:
that makes sense.  I'll resolved in the fb diff.